### PR TITLE
Create events when starting builds because of webhooks

### PR DESCRIPTION
**Pull request by tkyi**
- Renamed the webhook id from `eventId` to `hookId`
- Events are also created when a pull_request or push event webhook is sent

Follow up to https://github.com/screwdriver-cd/screwdriver/pull/337

### CODE REVIEW

File: plugins/webhooks/index.js [hunk 1]
```diff
@@ -300,10 +310,17 @@ function pushEvent(request, reply, parsed) {
...
 
-                    return buildFactory.create({ jobId, sha, username })
+                    return eventFactory.create({
+                        pipelineId,
+                        type: 'pipeline',
+                        workflow: pipeline.workflow,
+                        username,
+                        sha
+                    })
+                    .then(event => buildFactory.create({ jobId, sha, username, eventId: event.id }))
```
Comment (d2lam) [inline plugins/webhooks/index.js]: This can be in an outer scope.

File: plugins/webhooks/index.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ function stopJob(job) {
  * Run pull request's main job
  * @method startPRJob
  * @param  {Object}       options
- * @param  {String}       options.eventId       Unique ID for this GitHub event
+ * @param  {String}       options.hookId        Unique ID for this scm event
  * @param  {String}       options.pipelineId    Identifier for the Pipeline
  * @param  {String}       options.name          Name of the new job (PR-1)
  * @param  {String}       [options.sha]         specific SHA1 commit to start the build with
```
Comment (minz1027) [inline plugins/webhooks/index.js]: required to creat event

File: test/plugins/webhooks/index.test.js [hunk 3]
```diff
@@ -520,7 +557,7 @@ describe('github plugin test', () => {
                     parsed.action = 'closed';
                     reqHeaders = {
                         'x-github-event': 'pull_request',
-                        'x-github-delivery': 'eventId',
+                        'x-github-delivery': 'hookId',
```
Comment (minz1027) [inline test/plugins/webhooks/index.test.js]: parsed.hookId is better?

Comment (tkyi) [inline test/plugins/webhooks/index.test.js]: fixed

File: plugins/webhooks/index.js (hunk 2 above)
Comment (tkyi) [inline plugins/webhooks/index.js]: updated!

File: plugins/webhooks/index.js (hunk 1 above)
Comment (FenrirUnbound) [inline plugins/webhooks/index.js]: You can construct the `jobId` outside of this also

Comment (tkyi) [inline plugins/webhooks/index.js]: Oh I see. Fixed 👍

**Pull request merged**